tests/api2/test_007_pool.py

```diff
@@ -133,7 +168,8 @@ def test_07_setup_function(image):
         IMAGES[image] = f"/dev/{mdconfig_results['output'].strip()}s1"
 
 
-def test_08_import_msdosfs():
```

Comment:
Any test that is importing/mounting a non-ZFS filesystem needs to be skipped on TrueNAS Enterprise altogether as that's a non-supported operation.

Reply:
OK, I was just maintaining that test I am not sure of what it does.

Reply:
I move it out of the pool test and move it in pool_import and mark it to skip all pool import test on HA.